spec/models/instance_spec.rb

```diff
@@ -3,7 +3,10 @@
 RSpec.describe Instance, type: :model do
   it { is_expected.to have_many(:instance_users) }
   it { is_expected.to have_many(:users).through(:instance_users) }
-  it { is_expected.to have_many(:announcements).class_name(Instance::Announcement.name) }
+  it do
+    is_expected.to have_many(:announcements).class_name(Instance::Announcement.name).
```

Comment:
We are using shoulda matchers here, if use expect, the code will be like:
```ruby
  it 'has many announcements with descending order' do
    expect(subject).to have_many(:announcements).class_name(Instance::Announcement.name).
      order(valid_from: :desc)
  end
```
which looks a bit weird to me, cause the <code>subject</code> is defined implicitly by gem. It also not consistent, because we manually write a description string ( <code>'has many announcements with descending order'</code> ) for this test.

@lowjoel  what do you think ?